.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,25 @@
-<!--
-Please read and follow these instructions before creating and submitting a pull request:
+<!-- Thanks for contributing! -->
 
-- If you're fixing a bug, ensure you add unit tests to prove that it works.
-- Before adding a feature, it is best to create an issue explaining it first. It would save you some effort in case we don't consider it should be included in node-fetch.
-- If you are reporting a bug, adding failing units tests can be a good idea.
--->
+## Purpose
+...
 
-**What is the purpose of this pull request?**
+## Changes
+...
 
-- [ ] Documentation update
-- [ ] Bug fix
-- [ ] New feature
-- [ ] Other, please explain:
+## Additional information
+...
 
-**What changes did you make? (provide an overview)**
 
-**Which issue (if any) does this pull request address?**
+-------------------------------------------------------------------------------------------------
```

Comment:
I'm pretty sure this will do the same thing.

```suggestion
___
```

Reply:
It will, just thought it did look nicer when it was longer...

Reply:
The raw version of the readme should be optimised for usability instead of readability.